cfn-resources/stream-instance/README.md

```diff
@@ -0,0 +1,13 @@
+# MongoDB::Atlas::StreamInstance
```

Comment:
This main readme needs to be adjusted, we can follow other resources as an example.

Reply:
Adjusted it, let me know if it's okay now

Reply:
is the README.md currently automated? its content looks like quite straightforward.

Reply:
Currently it's not automated